bql/semantic/expression.go

```diff
@@ -89,7 +89,7 @@ func formatCell(c *table.Cell) (string, error) {
 	if c.S != nil {
 		formatted, err := literal.DefaultBuilder().Build(literal.Text, *c.S)
 		if err != nil {
-			return "", fmt.Errorf("could not build literal from string, received error: %v", err)
+			return "", fmt.Errorf("in formatCell, could not build literal from string - received error: %v", err)
```

Comment:
WDYT of:
could not build a literal from string while trying to formatCell, got error:

Reply:
Done.